routes/index.js

```diff
@@ -36,7 +36,6 @@ exports.index = function (req, res, next) {
 
 exports.loginHandler = function (req, res, next) {
   // Uncomment code below to create vulnerabilities - NoSQL injection
-/*
   if (validator.isEmail(req.body.username)) {
     User.find({ username: req.body.username, password: req.body.password }, function (err, users) {
```

Comment:
### <sub><img src="https://res.cloudinary.com/snyk/image/upload/r-d/scm-platform/snyk-pull-requests/severity-high.svg" width="20"/></sub>&nbsp; NoSQL Injection

Unsanitized input from the HTTP request body flows into find, where it is used in an NoSQL query. This may result in an NoSQL Injection vulnerability.

<h6>
<a href="https://cwe.mitre.org/data/definitions/943.html">CWE-943</a> | <a href="https://docs.snyk.io/manage-risk/prioritize-issues-for-fixing/priority-score">Priority score 816</a> | Line 40</h6>

<details>

<summary><b>Data flow: 6 steps</b></summary>

`Step 1` https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_oauth_nodejs-goof/blob/6173f06c2974d1dba72049f370452dd5fb07ba42/routes/index.js#L40

`Step 2` [routes/index.js#40](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_oauth_nodejs-goof/blob/6173f06c2974d1dba72049f370452dd5fb07ba42/routes/index.js#L40)

`Step 3` [routes/index.js#40](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_oauth_nodejs-goof/blob/6173f06c2974d1dba72049f370452dd5fb07ba42/routes/index.js#L40)

`Step 4` [routes/index.js#40](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_oauth_nodejs-goof/blob/6173f06c2974d1dba72049f370452dd5fb07ba42/routes/index.js#L40)

`Step 5` [routes/index.js#40](https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_oauth_nodejs-goof/blob/6173f06c2974d1dba72049f370452dd5fb07ba42/routes/index.js#L40)

`Step 6` https://github.com/trailblazing-dx/tiago_app.dev.snyk_github_oauth_nodejs-goof/blob/6173f06c2974d1dba72049f370452dd5fb07ba42/routes/index.js#L40

</details>